e2e/cypress/integration/integrations/custom_slash_commands/custom_slash_commands_spec.js

```diff
@@ -0,0 +1,257 @@
+        });
+
+        // # Cleanup command
+        deleteCommand(team1, trigger);
+    });
+
+    it('MM-T703/MM-704 Show custom slash command in autocomplete', () => {
+        // # Open slash command page
+        cy.visit(`/${team1.name}/integrations/commands/installed`);
+
+        // # Create new Slash command
+        addNewCommand(team1, trigger, commandURL);
+
+        // # Open slash command page
+        cy.visit(`/${team1.name}/integrations/commands/installed`);
+
+        // # Update autocomplete
+        cy.get(':nth-child(1) > .item-details > .d-flex > .item-actions > a > span').click();
+        cy.get('#autocomplete').click();
+        const hint = '[test-hint]';
+        cy.get('#autocompleteHint').type(hint);
+        const desc = 'Auto description';
+        cy.get(':nth-child(10) > .col-md-5 > #description').type(desc);
+        cy.get('#saveCommand').click();
+
+        // # Go back to home channel
+        cy.visit(`/${team1.name}/channels/town-square`);
+        cy.wait(TIMEOUTS.TWO_SEC);
+
+        // # Type slash
+        cy.get('#post_textbox', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').clear().type('/');
+        cy.wait(TIMEOUTS.TWO_SEC);
+
+        // * Verify that command is in the list
+        cy.contains(`${trigger}`);
+
+        // # Type full command
+        cy.get('#post_textbox', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').type(`${trigger}`);
+        cy.wait(TIMEOUTS.TWO_SEC);
+
+        // * Verify that autocomplete info is correct
+        cy.get('.slash-command__title').should('have.text', `${trigger} ${hint}`);
+        cy.get('.slash-command__desc').should('have.text', `${desc}`);
+
+        // # Open slash command page
+        cy.visit(`/${team1.name}/integrations/commands/installed`);
+
+        // # Remove autocomplete
+        cy.get(':nth-child(1) > .item-details > .d-flex > .item-actions > a > span').click();
+        cy.get('#autocomplete').click();
+        cy.get('#saveCommand').click();
+
+        // # Go back to home channel
+        cy.visit(`/${team1.name}/channels/town-square`);
+        cy.wait(TIMEOUTS.TWO_SEC);
+
+        // # Run slash command
+        cy.get('#post_textbox', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').clear().type('/');
+        cy.wait(TIMEOUTS.TWO_SEC);
+
+        // * Verify that command is not in the list
+        cy.contains(`${trigger}`).should('not.exist');
+
+        // # Cleanup command
+        deleteCommand(team1, trigger);
+    });
+});
+
+function deleteCommand(team, trigger) {
+    // # Open slash command page
+    cy.visit(`/${team.name}/integrations/commands/installed`);
+
+    // # Delete slash command
+    cy.get(':nth-child(1) > .item-details > .d-flex > :nth-child(1) > .item-details__trigger').contains(`/${trigger}`);
+    cy.get(':nth-child(1) > .item-details > .d-flex > .item-actions > :nth-child(3) > .color--link > span').click();
```

Comment:
Same as my previous comment. Is there any easier way to address these elements? If not, can you be a bit more clear what this selector does?